### PR TITLE
Testcafe

### PR DESCRIPTION
**Pull request by Argn0**

### CODE REVIEW

**Comment by howardchung** [comment]
LGTM, just sync latest to fix lockfile

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,22 @@
-sudo: false
+dist: trusty
+sudo: required
 language: node_js
 node_js:
 - '10'
+addons:
+  firefox: latest
```
Comment (howardchung) [inline .travis.yml]: are we testing on firefox?

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
+     - google-chrome
+    packages:
+     - google-chrome-stable fluxbox
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3
+  - fluxbox >/dev/null 2>&1 &
+before_install:
+  - stty cols 80
```
Comment (howardchung) [inline .travis.yml]: curious what this is for

File: .travis.yml (hunk 1 above)
Comment (Argn0) [reply .travis.yml]: no we only test on chrome. I think I should remove that

File: .travis.yml (hunk 2 above)
Comment (Argn0) [reply .travis.yml]: I'm not sure. I think it has something to do with unix shell settings. it's from the testcafe documentations

File: testcafe/tests/players.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+} from 'testcafe';
+import {
+  fixtureBeforeHook,
+  fixtureBeforeEachHook,
+  fixtureAfterHook,
+  fixtureRequestHooks,
+  host,
+} from './../testsUtility';
+
+const paths = [
```
Comment (howardchung) [inline testcafe/tests/players.js]: optional: I like putting all the test paths in one file

File: .travis.yml (hunk 1 above)
Comment (Argn0) [reply .travis.yml]: actually...I dont think we can remove it. the chrome stuff is part of it i think?

**Comment by Argn0** [comment]
fixed the merge conflict

**Pull request merged**